src/System.Net.WebSockets.Client/src/System/Net/WebSockets/ClientWebSocket.cs

```diff
@@ -177,24 +177,24 @@ public Task ConnectAsync(Uri uri, CancellationToken cancellationToken)
             {
                 string errorMessage = SR.Format(
                         SR.net_WebSockets_Argument_InvalidMessageType,
-                        "Close",
-                        "SendAsync",
-                        "Binary",
-                        "Text",
-                        "CloseOutputAsync");
+                        nameof(WebSocketMessageType.Close),
```

Comment:
"Close" is the one we have in Desktop. Asked because I wasn't sure if this would change the exception text.